Microsoft.Toolkit.Services/Services/Twitter/TwitterDataProvider.cs

```diff
@@ -311,24 +289,37 @@ public async Task<bool> LoginAsync()
         /// </summary>
         public void Logout()
         {
-            var twitterCredentials = PasswordCredential;
-            if (twitterCredentials != null)
+            var credential = _passwordManager.Get("TwitterAccessToken");
+            if (credential != null)
             {
-                _vault.Remove(twitterCredentials);
-                ApplicationData.Current.LocalSettings.Values["TwitterScreenName"] = null;
+                _passwordManager.Remove("TwitterAccessToken");
+                _storageManager.Set("TwitterScreenName", null);
                 UserScreenName = null;
             }
 
             LoggedIn = false;
         }
 
+#if WINRT
         /// <summary>
         /// Tweets a status update.
         /// </summary>
         /// <param name="tweet">Tweet text.</param>
         /// <param name="pictures">Pictures to attach to the tweet (up to 4).</param>
         /// <returns>Success or failure.</returns>
         public async Task<bool> TweetStatusAsync(string tweet, params IRandomAccessStream[] pictures)
```

Comment:
I think that this methods should be deprecated in favor of the .NetStandard (Stream) ones.